lia-ui/build.gradle

```diff
@@ -46,6 +48,10 @@ def getGitHash = { ->
     return stdout.toString().trim()
 }
 
+def getPublishVersionSuffix = { ->
```

Comment:
We should keep the code as sry as possible. Let's remove this method from both core and UI; and export it from the root gradle file, similar to `getGitHash()`